builder/dockerfile/parser/testfiles/escape_whitespace/Dockerfile

```diff
@@ -0,0 +1,5 @@
+  #escape = `
+
+FROM image
+ENV foo `
+ENV bar baz
```

Comment:
No, this doesn't change the test. Only thing we are testing is if the first line sets escape char or not.